src/keypad.js

```diff
@@ -121,22 +6,21 @@ function getPassword($, password) {
   const buttonToKeypad = button => {
     const $button = $(button)
 
-    // Get the style attribute
-    const btnStyle = $button.attr('style')
-
-    // Parse style attribute for b64 picture
-    const b64pic = btnStyle.match(/data:image\/([a-zA-Z]*);base64,([^)]*)/)[2]
+    // Get the img tag source
+    const btnImageSrc = $button.find('img').attr('src')
 
-    // Create a PNG image from the b64 picture
-    const pngImg = PNG.sync.read(Buffer.from(b64pic, 'base64'))
+    // Parse image tag for b64 picture
+    const b64pic = btnImageSrc.match(
+      /data:image\/([a-zA-Z+]*);base64,([^)]*)/
+    )[2]
 
-    // Extract the limits of the digits within the PNG image
-    const limits = getImageLimits(pngImg, { R: 255, G: 255, B: 255 })
+    // Convert base64 encoded image to SVG text description
+    const svgImg = Buffer.from(b64pic, 'base64').toString('utf-8')
 
-    // Find the digit from the limits and associate it to its key
+    // Extract digit from SVG text description
     const keypad = {
       key: $button.attr('data-matrix-key'),
-      digit: getDigit(limits)
+      digit: svgImg.match(/ id="(\d)"/)[1]
```

Comment:
`BaseKonnector: critical : Cannot read property '1' of null`

Now, SVG no longer contains the "id" attribute ... Boursorama has updated these images :'(